hail/python/hailtop/utils/utils.py

```diff
@@ -687,6 +687,14 @@ def is_transient_error(e):
     if (isinstance(e, aiohttp.ClientPayloadError)
             and e.args[0] == "Response payload is not completed"):
         return True
+    if (isinstance(e, aiohttp.ClientOSError)
+            and len(e.args) >= 2
+            and e.args[0] == 1
```

Comment:
This is hard to parse. Should it be `e.args[0] == '1'` as is `e.args[0]` a string or int? Can we just check for the message in `e.args[1]`?